content/advent-2018/go-arrow.md

```diff
@@ -0,0 +1,531 @@
++++
+author = ["Sebastien Binet"]
+title = "Go and Arrow: building blocks for data science"
+linktitle = "go-arrow"
+date = 2018-12-18T00:00:00Z
+series = ["Advent 2018"]
++++
+
+Today we will see how [Arrow](https://arrow.apache.org) could be useful for data science, or -- really -- a lot of analysis' workloads.
+
+## Lingua franca
+
+In [Data Science](https://en.wikipedia.org/wiki/Data_science) and in many scientific fields, the _lingua franca_ is Python.
+
+![xkcd-gravity](/postimages/advent-2018/go-arrow/xkcd-python.png)
+
+This means that a lot of the libraries are written in Python, with the CPU intensive parts leveraging either [NumPy](https://www.numpy.org/), [SciPy](https://www.scipy.org/) or, directly, a C/C++ library wrapped with the CPython C-API.
+This also means that:
+
+- the vast majority of the analysis pipelines are written in Python;
+- it is not completely straightforward to migrate parts of these pipelines to other languages, in an adiabatic and piecewise process.
```

Comment:
The word "adiabatic" was unfamiliar to me; I had to look it up. Maybe use a different word?